lib/rules/no-invalid-position-at-import-rule/__tests__/index.js

```diff
@@ -149,3 +149,56 @@ testRule({
 		},
 	],
 });
+
+testRule({
```

Comment:
Let's move this above the previous `testRule` as we tend to put non-standard tests last.